tests/openwisp2/settings.py

```diff
@@ -118,3 +162,5 @@
     TEMPLATES[0]['DIRS'].insert(
         0, os.path.join(BASE_DIR, 'sample_notifications', 'templates')
     )
+    # Celery auto detects tasks only from INSTALLED_APPS
+    CELERY_IMPORTS = ('openwisp_notifications.tasks',)
```

Comment:
mention this in the documentation as it's done in [openwisp-monitoring](https://github.com/openwisp/openwisp-monitoring#import-celery-tasks)